Makefile

```diff
@@ -21,35 +21,49 @@ ifeq (run,$(firstword $(MAKECMDGOALS)))
 endif
 
 .PHONY: all
+## Alias for `build`
 all: build
 
 .PHONY: build
+## Build runtime docker image
 build:
 	./scripts/build_docker.sh --target runtime
 
 .PHONY: build.develop
+## Build develop docker image
 build.develop:
 	./scripts/build_docker.sh --target develop
 
 .PHONY: develop
+## Build develop docker image and run an interactive shell in the develop envionment
 develop: build.develop
 	./scripts/develop.sh
 
 .PHONY: run
+## Build develop docker image and run a make command in the develop envionment (e.g. `make run fmt` will execute `make fmt` within the docker container)
 run: build.develop
 	./scripts/develop.sh make $(RUN_ARGS)
 
 .PHONY: test
+## Run tests
 test:
 	./scripts/run_tests.sh
 
 .PHONY: fmt
+## Run formatting
```

Comment:
might be good to tie this to the corresponding `lint` PR check?
```
## Auto-format source code and report code-style violations (lint)
```
or just
```
Format sources (lint)
```
?

Reply:
```suggestion
## Auto-format source code and report code-style violations (lint)
```